releasetasks/release_configs/dev_mozilla-release_fennec_rc_graph_2.yml

```diff
@@ -23,7 +23,8 @@ update_verify_enabled: false
 push_to_releases_enabled: true
 # rc graph 2 only needs final_verify_channels
 channels:
-publish_to_balrog_channels: []
+publish_to_balrog_channels:
+    - "release-dev"
```

Comment:
We use "*-dev" in case we mess up the configs somewhere and try to post to the prod instance. Last year, before we switched to staging balrog, we had a an ec2 balrog instance, where the channel was configured. In any case, this shouldn't be an issue.